lib/common.js

```diff
@@ -545,6 +574,7 @@ exports.readMysqlValue = function(parser, column, columnSchema) {
     case MysqlTypes.DECIMAL: // Deprecated in MySQL > 5.0.3
     case MysqlTypes.NEWDATE: // Not used
     default:
+      // TODO: Remove throw statement to make module safe
```

Comment:
This error can be occur multiple times and can be emitted from the ZongJi object in `lib/rows_events.js` with a `try...catch` around line 111.